src/test/java/core/basesyntax/figure/figures/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figure.figures;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.interfaces.Area;
+import core.basesyntax.suppliers.ColorSupplier;
+import java.util.Random;
+
+public class Circle extends Figure implements Area {
+    private final Random random = new Random();
+    private static final int RADIUS_DEFAULT = 10;
+    private static final String COLOR_DEFAULT = "DefaultColorCircle";
+    private static final String NAME_DEFAULT = "CIRCLE";
```

Comment:
let's remove such constants, they don't need you, you can just add name of the figure directly in draw()